tasks/e2e.sh

```diff
@@ -28,6 +28,30 @@ function handle_exit {
   exit
 }
 
+function build_check {
+  local test_snap_path="$1"
+  if [ -z "$1" ]
```

Comment:
Yes, we always passed in snap file path in the 3 tests.

I am thinking what the `build_check` should behave when it is called without given $1.

-- option 1: delete the if check, let the function fall and let developer knows that Jest test is mandatory
-- option 2: use the following code, skip Jest test when there is no $1

```
if [ -z "$1" ]
    then
      test -e $1
fi
```

I will update the code when received your suggestion.